### PR TITLE
fix: handle early deposits

### PR DESCRIPTION
**Pull request by mrice32**

### CODE REVIEW

File: src/utils/DepositUtils.ts [hunk 1]
```diff
@@ -93,6 +93,32 @@ export function getUniqueDepositsInRange(
...
+  originClient: SpokePoolClient,
+  existingUniqueDeposits: typechain.FundsDepositedEvent[]
+): typechain.FundsDepositedEvent[] {
+  const originChainBlockRange = getBlockRangeForChain(
+    blockRangesForChains,
+    originChain,
+    chainIdListForBundleEvaluationBlockNumbers
+  );
+  return (originClient["earlyDeposits"] as unknown as typechain.FundsDepositedEvent[]).filter(
+    (deposit: typechain.FundsDepositedEvent) =>
```
Comment (james-a-morris) [inline src/utils/DepositUtils.ts:110]: Nit: with the typecast above, do we need to specify this function param?

**Review by james-a-morris — APPROVED** [review]
Left one nit. I helped with typecasting & unit tests so a second approval may be in order. Otherwise, this code seems good & it passes on my integration tests locally

File: src/utils/DepositUtils.ts (hunk 1 above)
Comment (mrice32) [reply src/utils/DepositUtils.ts:110]: @james-a-morris maybe not, but just in the interest of time, let's do this in a follow-up.

**Comment by james-a-morris** [comment]
LGTM

**Pull request merged**